sycl/doc/extensions/supported/sycl_ext_intel_esimd/examples/README.md

```diff
@@ -73,7 +73,8 @@ is to show the basic ESIMD APIs in well known examples.
      [[intel::device_indirectly_callable]] SYCL_EXTERNAL void __regcall invoke_slm_load_store(
      local_accessor<int, 1> *local_acc, uint32_t slm_byte_offset, int *in, int *out,
      simd<uint32_t, VL> global_byte_offsets) SYCL_ESIMD_FUNCTION {
-        esimd::simd<uint32_t, VL> esimd_global_byte_offsets = global_byte_offsets;
+      esimd::simd<uint32_t, VL> esimd_global_byte_offsets = global_byte_offsets;
+      // Modify SLM in ESIMD context.
```

Comment:
minor comment: this is Read of SLM in ESIMD context, not 'Modify'.

Reply:
ah yeah sorry